rest-api/src/main/java/com/cloudbees/workflow/rest/external/RunExt.java

```diff
@@ -189,7 +189,10 @@ public void setArtifacts(Link artifacts) {
 
     /** Computes timings after the stages have been set up
      *  That means timing of the stage has been computed, and the stages are sorted
+     *
+     *  Deprecated but retained for external APIs consuming it, use {@link #createNew(WorkflowRun)} instead
```

Comment:
@jglick I can understand the confusion -- this is a case where the diff display is being completely horrible.  I had to expand all the sections to confirm something hadn't gone amiss, but the actual logic changes are in `createNew` (and Github "conveniently" collapses the end of `computeTimings` and the beginning of `createNew`).

It's clearer if you check it out and pull it up in meld or another diff tool. 

You're right that this should not be public.  But as far as `@Restricted` -- my concern is that someone's plugin *might* be invoking this in another plugin (library API, not REST) -- which that annotation would break. 

Tempting to just remove the method and risk it though, since consumers are limited and someone *should not* be using this method anyway.